cli/tests/containerize.bats

```diff
@@ -104,24 +104,6 @@ function skip_if_linux() {
   assert_output --partial "'containerize' is currently only supported on linux (found macos)."
 }
 
-# bats test_tags=containerize:default-to-file
-@test "container is written to a file by default" {
```

Comment:
The codepath for containerizing a v0 env is different than a v1 env, so I think it might be good to keep this one test to exercise that codepath. But I think we could drop all the other cases. If we keep this 1 and drop the rest, would that still save a good chunk of time?